### PR TITLE
node and text reps should always pass grips to callbacks

### PR DESCRIPTION
**Pull request by juliandescottes**

### CODE REVIEW

**Comment by juliandescottes** [comment]
As discussed, here's a patch to:
- expect an array of actor ids that are in tree
- always pass grips to callbacks 

Note that the array is not mandatory here, in order to remain backward compatible. If the array is not provided, we consider any Node/Text element to be in tree.

File: src/test/mochitest/test_reps_grip-map.html [hunk 1]
```diff
@@ -252,11 +252,11 @@
 
     const valuesGrips = getSelectableInInspectorGrips(nodeValuedStub);
     is(valuesGrips.length, 3, "the stub has three node grips");
-    const valuesAttachedNodeFrontsByActor = getStubNodeFront(valuesGrips);
+    const valuesattachedActorIds = getStubAttachedActorIds(valuesGrips);
```
Comment (nchevobbe) [inline src/test/mochitest/test_reps_grip-map.html]: nit: valuesAttachedActorIds (camelCase)

File: src/test/mochitest/test_reps_grip-map.html [hunk 2]
```diff
@@ -252,11 +252,11 @@
...
 
     const keysGrips = getSelectableInInspectorGrips(nodeKeyedStub);
     is(keysGrips.length, 3, "the stub has three node grips");
-    const keysAttachedNodeFrontsByActor = getStubNodeFront(keysGrips);
+    const keysattachedActorIds = getStubAttachedActorIds(keysGrips);
```
Comment (nchevobbe) [inline src/test/mochitest/test_reps_grip-map.html]: nit: camelCase

Comment (juliandescottes) [inline src/test/mochitest/test_reps_grip-map.html]: "Did someone use replace all here or what?" :)

**Comment by juliandescottes** [comment]
Just merged the CI update on master and rebased this branch, let's wait for the test results!

**Comment by juliandescottes** [comment]
CI fixed let's merge!

**Pull request merged**